src/Sets/SparsePolynomialZonotope.jl

```diff
@@ -350,31 +350,3 @@ function remove_redundant_generators(S::SparsePolynomialZonotope)
     GI = remove_zero_columns(genmat_indep(S))
     return SparsePolynomialZonotope(c, G, GI, E)
 end
-
-# function quadratic_map(Q::Vector{MT}, S::SparsePolynomialZonotope) where {N, MT<:AbstractMatrix{N}}
```

Comment:
Why did you remove this method?

Reply:
this was a copy-paste of the code for SSPZ (note it was commented out), meant to be a starting point for the implementation for SPZ. Since the method computes an overapproximation, I understood the right way is to define `overapproximate(::QuadraticMap, ::Type{SparsePolynomialZonotope})` as I did in this PR. Should I also have a `quadratic_map(Q, S::SparsePolynomialZonotope)` ?

Reply:
Ah okay, I did not see that it was commented out. Then it is fine.
No, we do not want to use `quadratic_map` for overapproximations.